lib/pages/movies_page.dart

```diff
@@ -0,0 +1,176 @@
+              children: <Widget>[
+                MoviePoster(
+                  pathToPosterImg: movie.pathToPosterImg,
+                  width: MoviesPage.posterWidth,
+                  height: MoviesPage.posterHeight,
+                ),
+                Container(width: MoviesPage.imageSpacing),
+                Expanded(
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: <Widget>[
+                      Container(height: MoviesPage.textSpacing),
+                      Text(
+                        movie.title,
+                        style: const TextStyle(
+                          fontSize: MoviesPage.titleFontSize,
+                          color: Colors.white,
+                          fontWeight: FontWeight.bold,
+                        ),
+                      ),
+                      Container(height: MoviesPage.textSpacing),
+                      Text(
+                        releaseYear,
+                        style: TextStyle(
+                          color: Colors.grey[300],
+                        ),
+                      ),
+                      Container(height: MoviesPage.textSpacing),
+                      Wrap(
+                        children: [
+                          for (String genre in movie.genres
+                              .take(MoviesPage.displayedGenresAmount))
+                            _buildGenreCard(genre),
+                        ],
+                      ),
+                      Center(
+                        child: Transform.scale(
+                          scale: MoviesPage.scoreIconScale,
+                          child: MovieScore(
+                            movieScore: movie.score,
+                            voteCount: movie.voteCount,
+                          ),
+                        ),
+                      ),
+                    ],
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Card _buildGenreCard(String genre) {
+    return Card(
+      color: UIConstants.genreCardColor,
+      child: Container(
+        decoration: BoxDecoration(
+          borderRadius: const BorderRadius.all(
+            Radius.circular(UIConstants.genreCardBorderRadius),
+          ),
+          border: Border.all(
+            color: UIConstants.genreCardBorderColor,
+            width: UIConstants.genreCardBorderWidth,
+          ),
+        ),
+        child: Padding(
+          padding: const EdgeInsets.all(MoviesPage.genreCardPadding),
+          child: Text(
+            genre,
+            style: const TextStyle(
+              color: UIConstants.genreCardTextColor,
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return FutureBuilder(
+      future: movies,
+      builder: (BuildContext context, AsyncSnapshot snapshot) {
```

Comment:
trailing comma

Reply:
And also you can specify on AsyncSnapshot the data that you are expecting like List<Movie> in this case, to avoid cast on line 145

Reply:
good tip, thanks!